r/vignettes/install.Rmd

```diff
@@ -28,35 +28,6 @@ For `gcc`, this generally means version 7 or newer. Most contemporary Linux
 distributions have a new enough compiler; however, CentOS 7 is a notable
 exception, as it ships with gcc 4.8.
 
-If you are on CentOS 7, to build arrow you will need to install a newer `devtoolset`, and you'll need to update R's Makevars to define the `CXX17` variables. This script installs `devtoolset-8` and configures R to be able to use C++17:
```

Comment:
The note in the paragraph above also mentions CentOS 7. 

Maybe we should keep this discussion and say something like, we don't officially support CentOS 7 anymore, but if you want to try building on it, you'll need to:

Reply:
_nods_ ok yeah, I also like that it leaves a bit of a more obvious breadcrumb trail for folks debugging. I know it's in the git history, but I don't think folks would know to look here in that history necessarily to find out "what do I do with non-standard compiler setups?"